pkg/webservice/handlers.go

```diff
@@ -279,6 +280,7 @@ func GetContainerHistory(w http.ResponseWriter, r *http.Request) {
 		if err := json.NewEncoder(w).Encode(result); err != nil {
 			http.Error(w, err.Error(), http.StatusInternalServerError)
 		}
+		return
```

Comment:
This should be the equivalent of what we have for application history

Reply:
Sorry, I missed it. Fixed.